src/index.js

```diff
@@ -1,94 +1,184 @@
 import F from 'futil'
 import _ from 'lodash/fp'
-import { observable, extendObservable, toJS } from 'mobx'
+import { observable, extendObservable } from 'mobx'
 import * as validators from './validators'
+import { tokenizePath, pickFields, safeJoinPaths } from './util'
+import { treePath, splitAt } from './futil'
+import { get, set, toJS } from './mobx'
+export { validators }
 
+let clone = _.flow(toJS, _.cloneDeep)
 let unmerge = _.flow(F.simpleDiff, _.mapValues('to'))
 let changed = (x, y) => !_.isEqual(x, y) && !(F.isBlank(x) && F.isBlank(y))
 let Command = F.aspects.command(x => y => extendObservable(y, x))
+let throwIfLeaf = x =>
+  !_.has('fields', x) &&
+  !_.has('itemField', x) &&
+  F.throws(`${_.join('.', x.path)}: No fields or itemField`)
+let fieldPath = _.flow(F.intersperse('fields'), _.compact)
 
-let values = _.mapValues('value')
-let Form = ({
+export default ({
   afterInitField = x => x,
   validate = validators.functions,
   ...config
 }) => {
-  let initField = (x, field) => {
+  let saved = {}
+
+  let state = observable({
+    // config.field values are more of default values than anything else
```

Comment:
this is confusing because you'd expect more specific `value`s to take precedent over a top level value

Reply:
after talking again about `defaultValue` on array items, there is good reason for this behavior but we should document it )(because parent values should take precedent for arrays because the children values would be defaults and not have a value for each child)